pkg/sql/plan/function/builtin/unary/math.go

```diff
@@ -85,8 +86,24 @@ func Ln(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
 }
 
 func Log(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
-	// MySQL log is the same as ln.
-	return math1(vs, proc, momath.Ln)
+	if len(vs) == 1 {
```

Comment:
No, this is the wrong fix.   We never introduce null to a math function.   Error is error.

Reply:
Because the log function can use two parameters, the main purpose here is to distinguish between one parameter and two parameters